cpp-terminal/base.cpp

```diff
@@ -33,39 +33,39 @@ void Term::write(const std::string& s) {
     std::cout << s << std::flush;
 }
 
-std::string Term::cursor_off() {
+const char* Term::cursor_off() {
     return "\x1b[?25l";
 }
```

Comment:
We should not use `char*` in C++, unless we have to. Let's use `std::string`.

Reply:
I just thought we can skip ther conversion. The rueturn is actually a const char pointer after all. But I'll revert that one as well.